### PR TITLE
Address OWASP findings

### PR DESCRIPTION
**Pull request by SmithJosh**
#### What does this PR do?
Upgrades Jolokia to 1.6.2, Hazelcast to 3.12.10, suppresses a false positive, and switches to jetty-bom to force the jetty version.

#### Select relevant component teams: 
@codice/security 

#### How should this be tested?

CI

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by SmithJosh** [comment]
build now

**Review by vinamartin — APPROVED** [review]

**Review by emmberk — APPROVED** [review]

**Review by jlcsmith — APPROVED** [review]

File: dependency-check-maven-config.xml [hunk 1]
```diff
@@ -150,4 +150,21 @@
         <cve>CVE-2020-11980</cve>
     </suppress>
 
+    <suppress>
+        <notes>
+            Zookeeper is the only thing still using Log4J, and it has suppressions for both the
```
Comment (shaundmorris) [inline dependency-check-maven-config.xml:155]: Nice notes. We're using SLF4J, correct? 👍

**Review by shaundmorris — APPROVED** [review]

File: dependency-check-maven-config.xml (hunk 1 above)
Comment (SmithJosh) [reply dependency-check-maven-config.xml:155]: We mostly use the SLF4J api, though there's a few instances where we use others (Pax logging lets you use any of the standard logging apis). But the logging implementation we use is Log4J2

**Comment by SmithJosh** [comment]
Rebased. build now

**Comment by SmithJosh** [comment]
Failed due to flakey itest. build now

**Comment by SmithJosh** [comment]
build now

**Comment by SmithJosh** [comment]
Rebased. build now

**Comment by SmithJosh** [comment]
Rebased again. build now

**Comment by SmithJosh** [comment]
Unrelated itest failure:
```
[2021-01-12T15:44:22.693Z] [ERROR] Failures: 
[2021-01-12T15:44:22.693Z] [ERROR]   TestFederation.testFederatedNegativeSpatial Failed to setup exam for TestFederation: java.lang.AssertionError: Failed in @BeforeSuite: java.lang.IllegalStateException: Failed to start up required features.
```

build now

**Pull request merged**